ui/src/main/js/application/auth/SamlForm.js

```diff
@@ -90,6 +97,20 @@ const SamlForm = ({ csrfToken, errorHandler, readonly, displayTest, fileDelete,
                 submitLabel="Save SAML Configuration"
                 testLabel="Test SAML Configuration"
             >
+                <LabeledField label={importBlackDuckSSOConfigLabel} description={importBlackDuckSSOConfigDescription}>
+                    <div className="d-inline-flex p-2">
+                        <GeneralButton id="blackduck-sso-import-button" onClick={() => setShowBlackDuckSSOImportModal(true)}>Fill Form</GeneralButton>
```

Comment:
Add in a `disabled={!samlEnabled}` flag to this.  It will disable the button unless the user has enabled SAML authentication via clicking the checkbox in line 114.  
  
```
<GeneralButton 
    id="blackduck-sso-import-button" 
    onClick={() => setShowBlackDuckSSOImportModal(true)}
    disabled={!samlEnabled}
>Fill Form</GeneralButton>
```